src/ui/popup/settings-page/devtools.tsx

```diff
@@ -25,15 +38,25 @@ function getExistingDevToolsWindow() {
 
 async function openDevTools() {
     const devToolsWindow = await getExistingDevToolsWindow();
-    if (devToolsWindow) {
-        chrome.windows.update(devToolsWindow.id, {'focused': true});
+    if (!isMobile()) {
+        if (devToolsWindow) {
+            chrome.windows.update(devToolsWindow.id, {'focused': true});
+        } else {
+            chrome.windows.create({
+                type: 'popup',
+                url: isFirefox() ? '../devtools/index.html' : 'ui/devtools/index.html',
+                width: 600,
+                height: 600,
+            });
+        }
     } else {
-        chrome.windows.create({
-            type: 'popup',
-            url: isFirefox() ? '../devtools/index.html' : 'ui/devtools/index.html',
-            width: 600,
-            height: 600,
-        });
+        if (devToolsWindow) {
+            chrome.tabs.update(devToolsWindow.id, {'active': true});
+        } else {
+            chrome.tabs.create({
+                url: '../devtools/index.html'
```

Comment:
Extensions also work on mobile Chromium browsers like Kiwi, so use `getDevToolsURL()` function here.

Reply:
What should the function do?

Reply:
for isMobile i guess...

Reply:
I meant the function should open return the URL the correct devtools page for Chrome in Firefox, they are different. When there's some strange logic repeated multiple times, it's better to move it into a function. But if you managed to fix the issue without it, then leave as is.

Reply:
Firefox mobile now fixed will look for kiwi browser